demo/vmdefs/available/centos6-wordpress/Vagrantfile

```diff
@@ -0,0 +1,32 @@
+VAGRANTFILE_API_VERSION = "2"
+N = 2 
```

Comment:
nah, is it possible to create hash with hostname and ip and then iterate over that? I don't like the idea how you calculate ip and set hostname.

From tips&trick site (https://www.vagrantup.com/docs/vagrantfile/tips.html)
```The Vagrantfile is a very flexible configuration format. Since it is just Ruby, there is a lot you can do with it. However, in that same vein, since it is Ruby, there are a lot of ways you can shoot yourself in the foot. When using some of the tips and tricks on this page, please take care to use them correctly.```

Reply:
I see it comes from https://www.vagrantup.com/docs/provisioning/ansible.html#ansible-parallel-execution but still I think we should change that. What others think?